types/relay-runtime/lib/handlers/connection/RelayConnectionHandler.d.ts

```diff
@@ -14,29 +13,22 @@ export interface ConnectionMetadata {
     count: string | null | undefined;
 }
 
-export function buildConnectionEdge(
-    store: RecordSourceProxy,
-    connection: RecordProxy,
-    edge: RecordProxy | null | undefined,
-): RecordProxy | null | undefined;
+export const RelayDefaultHandlerProvider: {
+    buildConnectionEdge(
+        store: RecordSourceProxy,
+        connection: RecordProxy,
+        edge: RecordProxy | null | undefined,
+    ): RecordProxy | null | undefined;
 
-export function createEdge(
-    store: RecordSourceProxy,
-    record: RecordProxy,
-    node: RecordProxy,
-    edgeType: string,
-): RecordProxy;
+    createEdge(store: RecordSourceProxy, record: RecordProxy, node: RecordProxy, edgeType: string): RecordProxy;
 
-export function deleteNode(record: RecordProxy, nodeID: DataID): void;
+    deleteNode(record: RecordProxy, nodeID: DataID): void;
 
-export function getConnection(
-    record: ReadonlyRecordProxy,
-    key: string,
-    filters?: Variables | null,
-): RecordProxy | null | undefined;
+    getConnection(record: ReadOnlyRecordProxy, key: string, filters?: Variables | null): RecordProxy | null | undefined;
 
-export function insertEdgeAfter(record: RecordProxy, newEdge: RecordProxy, cursor?: string | null): void;
+    insertEdgeAfter(record: RecordProxy, newEdge: RecordProxy, cursor?: string | null): void;
 
-export function insertEdgeBefore(record: RecordProxy, newEdge: RecordProxy, cursor?: string | null): void;
+    insertEdgeBefore(record: RecordProxy, newEdge: RecordProxy, cursor?: string | null): void;
 
-export function update(store: RecordSourceProxy, payload: HandleFieldPayload): void;
+    update(store: RecordSourceProxy, payload: HandleFieldPayload): void;
+};
```

Comment:
These functions should not be exported as part of an interface, but rather as separate functions, as was done before. See how they are exported individually here? https://github.com/facebook/relay/blob/7253d354d31786488e6446dee10516fe545c9454/packages/relay-runtime/handlers/connection/RelayConnectionHandler.js#L538-L546

Reply:
Sent you a PR https://github.com/renanmav/DefinitelyTyped/pull/1

Reply:
Perhaps this needs some test coverage, seeing as it’s an odd one out. Care to add some?